src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -396,6 +396,8 @@
     <!-- Non-null type is required for this node kind -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
 
+    <Field Name="IsDeclaration" Type="bool"/>
```

Comment:
What does this field represent? Don't we have a separate BoundLocalDeconstructionDeclaration node to represent declarations?